### ISSUE TITLE
Problem installaing aRchaic

### ISSUE THREAD

**Issue by thierrygrange** [kkdey/aRchaic#5]
I could not install aRchaic following the instructions described on the github page. Here is the error message I received:
Error : object ‘get_viewport_logo’ is not exported by 'namespace:Logolas'
ERROR: lazy loading failed for package ‘aRchaic’
* removing ‘/home/thierry/R/x86_64-pc-linux-gnu-library/3.4/aRchaic’
Installation failed: Command failed (1)
Any idea about the solution? 
Thanks
Thierry

**Comment by rsh249** [comment]
You may need R 3.5. In my experience the Logolas package does not install properly on older versions. Check if Logolas is even installed and/or upgrade R.

**Comment by thierrygrange** [comment]
OK, updating to R 3.5 did solve the problem. Since R 3.4 is the default version in Ubuntu at the moment, I suggest that you mention that, for Ubuntu user, one should add/modify the  /etc/apt/sources.list file to include
deb https://cloud.r-project.org/bin/linux/ubuntu bionic-cran35/ 
with the address corresponding to the preferred CRAN mirror and the Ubuntu version being adapted to that of the computer bionic/xenial/trusty that are the versions supported for R 3.5, that may save time to others.

**Comment by rsh249** [comment]
3.4 is the default on many Mac OS setups as well.

**Comment by thierrygrange** [comment]
Thanks for your help Rob. I notice that you are not on the list of the authors. You solved my problem though. Thanks for your involvement. And for the authors, please update your installation instructions to guide Ubuntu (and Mac as well apparently) users to use R3.5 that seems to be a requirement.
Thierry

**Comment by kkdey** [comment]
Yes,  aRchaic requires R version >=3.5.  We did mention it in the vignette but perhaps it was not very clear...we have updated the installation information in the vignette and the webpage and added the info to the README as well. Thanks @rsh249 for helping out with the information.

**Comment by rsh249** [comment]
No problem.
 
I think the issue comes up not because of a lack of documentation, but rather because most people use R3.4 still and because everything except for the Logolas package installs fine in the aRchaic installation.

**Comment by kkdey** [comment]
Thanks @rsh249 for that heads up. We are actually having an issue with fixing the old version (R>=3.4) of Logolas on Bioconductor and we hope to be able to fix that soon.

In the meantime, I have now updated the README to point the user with R(>=3.4) to install Logolas via Github instead of installing it from Bioc. I hope that can bypass the installation issue for the users
